mava/configs/env/gigastep.yaml

```diff
@@ -0,0 +1,11 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: hide_and_seek_5_vs_5 # [hide_and_seek_5_vs_5, identical_5_vs_5, waypoint_5_vs_5]
```

Comment:
I thought that having the option to customize the environment would be better since the base scenarios are a bit limiting and they only go up to 20v20 agents (the more intense scenarios are commented out) and they dont allow for any form of reward control

Reply:
Hmmm I see, honestly it's an environment issue, let's leave it for now and later we'll fix it with the other config scenarios. Can you add comments in the configs for more clarity and know what to change when we create a new task config (to keep the other flags fixed).

Also add the name of env and task_name there(see lbf example).